src/hotspot/cpu/aarch64/register_aarch64.hpp

```diff
@@ -69,8 +69,8 @@ class Register {
 
   constexpr Register() : _encoding(-1) {} // noreg
 
-  int operator==(const Register r) const { return _encoding == r._encoding; }
-  int operator!=(const Register r) const { return _encoding != r._encoding; }
+  constexpr bool operator==(const Register r) const { return _encoding == r._encoding; }
+  constexpr bool operator!=(const Register r) const { return _encoding != r._encoding; }
```

Comment:
This seems unrelated to the rest of this change.  It also seems like something that should be done for all
of the register_<cpu> variants.

Reply:
It was related to another reviewer's comments, but we don't need it